MaterialLibrary/src/main/java/io/doist/material/drawable/WrapperDrawable.java

```diff
@@ -142,6 +143,11 @@ protected void onBoundsChange(Rect bounds) {
     }
 
     @Override
+    public void setBounds(@NonNull Rect bounds) {
```

Comment:
Nope, it was a really bad fix for an issue 😓 Found a proper one and will remove this commit.